crates/project/src/dap_store.rs

```diff
@@ -6,9 +7,14 @@ use dap::SourceBreakpoint;
 use gpui::{EventEmitter, ModelContext, Task};
 use language::{Buffer, BufferSnapshot};
 use settings::WorktreeId;
+use sqlez::{
```

Comment:
Im not sure if this the right place to import sqlez. Feel like this belongs more inside the workspace.

Reply:
I think we should talk about this with the Zed team, because I've been finding that I have to redefine serialize breakpoint's in workspace and would rather have all the breakpoints in dap_store. On the other hand, keeping the database related stuff in workspace makes sense, so it's going to be a trade off no matter what.

Reply:
Note: We talked with the zed team and will be moving breakpoint code to workspace in the near future